src/platform/packages/shared/kbn-esql-validation-autocomplete/src/shared/context.ts

```diff
@@ -179,19 +183,24 @@ export function getAstContext(queryString: string, ast: ESQLAst, offset: number)
       // command ... "<here>"
       return { type: 'value' as const, command, node, option, containingFunction };
     }
+    // /**
```

Comment:
Why is this commented out?

Reply:
leftover, will remove

Reply:
Updated in https://github.com/elastic/kibana/pull/216379/commits/866a5e41e331836150f1f575b1aaa279b2f0764f